### PR TITLE
Style link as a button and force text to be white

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd & @Stanford-Online/openedx-courseops

example of what it will look like:

screen shot 2016-08-29 at 3 23 49 pm

### CODE REVIEW

File: qualtricssurvey/public/view.less [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .qualtricssurvey_block {
-    // TODO: Put your styles here
+    .qualtrics-button {
+        color: white !important;
```
Comment (stvstnfrd) [inline qualtricssurvey/public/view.less]: Which selector overrides this without `!important`?

Comment (caesar2164) [reply qualtricssurvey/public/view.less]: there's a bunch of selectors in platform that sets `a:visited` to a dark color, which looks funky. I guess I can just make my selector `.qualtrics-button:visited`

**Comment by stvstnfrd** [comment]
@caesar2164 Were you there when @dcadams explained how he runs these tests locally?

I know it was some form of `/edx/bin/python.edxapp manage.py lms SOMETHING` but I can't remember the exact invocation.

@caesar2164 Can you run through these and see where we stand (and touch base with David, if needed)?

**Comment by caesar2164** [comment]
From hipchat:

> To run the qualtrics survey unit tests:
> 
> Download the repo into a local virtual env on your box
> Rig your github.txt file in edx-platform to point to that virt env
> Restart your edx-platform server so the xblock gets installed
> Go into you edx-platform virtual env and run:    ./manage.py lms test --settings=test qualtricssurvey

**Comment by stvstnfrd** [comment]
Thanks for the info @caesar2164 . Let me know how the tests go.

**Comment by stvstnfrd** [comment]
Assuming tests passed, I'm +1.

**Pull request merged**